src/components/RadioPicker/styles.scss

```diff
@@ -34,7 +34,7 @@
       z-index: -1;
 
       &:checked + label {
-        --RadioPicker-color-text: var(--sds-clr-base-00);
+        --RadioPicker-color-text: var(--sds-clr-white);
```

Comment:
Fixing this to have a better contrast in dark mode (white text on purple instead of black text on purple).